cmd/cmd.go

```diff
@@ -896,6 +898,11 @@ func checkServerHeartbeat(_ *cobra.Command, _ []string) error {
 func NewCLI() *cobra.Command {
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
 
+	if runtime.GOOS == "windows" {
```

Comment:
This looks like an aesthetic change unrelated to the CUDA support, correct? There's gonna be a few things we need to fix in the CLI for an official windows release, so I think this would make more sense as a separate PR.